internal/pkg/manifest/validate.go

```diff
@@ -850,6 +850,28 @@ func (a AdvancedCount) Validate() error {
 		}
 	}
 
+	// Validate combinations with cooldown
+	if a.CPU != nil {
```

Comment:
Nice validation. Also we probably can simplify the logic by checking if `a.cooldown` is set first. If it is not set then we don't even need to go through each advanced scaling policy setting.